ping-server.js

```diff
@@ -0,0 +1,44 @@
+const prefs = require('sdk/simple-prefs').prefs;
+const request = require('sdk/request').Request;
+const getCookiesFromHost = require('./cookie-manager');
+
+function pingServer(title, addon) {
+  getCookiesFromHost(prefs.HOSTNAME, function(cookies) {
+    const headers = {};
+
+    cookies.forEach(function(c) {
+      headers.Cookie += c.name + '=' + c.value + ' ';
+      if (c.name === 'csrftoken') headers['X-CSRFToken'] = c.value;
+    });
+
+    getMe(headers, function(response) {
+      request({
+        url: 'http://' + prefs.HOSTNAME + '/api/metrics/?format=json',
```

Comment:
Ah, I think some of my comments got lost in the shuffle.

It would be good to avoid hard-coding `http://` throughout. Would you mind adding another pref that includes both the hostname and the `http://` part of the URL, so that we can easily switch to `https` when we test against production?